.github/workflows/alert-community.yml

```diff
@@ -13,28 +13,113 @@ jobs:
         with:
           files: "docs/*/*"
           reviewers: "@prince-chrismc"
-        
+
       - uses: ./.github/actions/alert-community
         with:
           files: ".github/*/*"
           reviewers: "@ericLemanissier @prince-chrismc"
-        
+
       - uses: ./.github/actions/alert-community
         with:
           files: "linter/*/*"
           reviewers: "@ericLemanissier @prince-chrismc"
-          
+
       - uses: ./.github/actions/alert-community
         with:
           files: "recipes/aaf/*/*"
           reviewers: "@MartinDelille"
-          
+
+      - uses: ./.github/actions/alert-community
+        with:
+          files: "recipes/avahi/*/*"
+          reviewers: "@jwillikers"
+
+      - uses: ./.github/actions/alert-community
+        with:
+          files: "recipes/boost/*/*"
```

Comment:
@grafikrobot Psst, you might want to look into this ;)

Reply:
That's exactly how it works.


I want to start small to see how people like it before investing too much effort but I like your idea!